client/webserver/site/src/html/dexsettings.tmpl

```diff
@@ -0,0 +1,41 @@
+{{define "dexsettings"}}
+{{template "top" .}}
+{{$passwordIsCached := .UserInfo.PasswordIsCached}}
+<div id="main" data-handler="dexsettings" data-host="{{.Exchange.Host}}" class="text-center py-5 overflow-y-auto">
+  <span class="settings-gear ico-settings"></span>
+  <h2 class="text-break">{{.Exchange.Host}}</h2>
+  <div class="d-flex justify-content-center">
+    <span class="fs20 mx-2 connection ico-connection d-hide" id="connectedIcon"></span>
+    <span class="fs20 mx-2 disconnected ico-disconnected d-hide" id="disconnectedIcon"></span>
+    <h4 id="connectionStatus"></h4>
+  </div>
```

Comment:
Need an `align-items-center` on the connection status div. Might as well just use `flex-center`. Proper alignment is also hindered by the `margin-bottom: 0.5rem;` that bootstrap adds to `h4`.

![image](https://user-images.githubusercontent.com/6109680/169641467-1c48a381-f7c2-4d0e-b6af-e72f047ed0cf.png)

The font sizes used are pretty large. Without a compelling reason to go larger, `.fs28` is the largest font-size I'd like to use throughout.

```html
 <div class="flex-center fs28">{{.Exchange.Host}}</div>
 <div class="flex-center fs16 mb-2">
    <span class="me-2 connection ico-connection d-hide" id="connectedIcon"></span>
    <span class="disconnected ico-disconnected d-hide" id="disconnectedIcon"></span>
    <span id="connectionStatus"></span>
 </div>
```
![image](https://user-images.githubusercontent.com/6109680/169641455-7922ec9d-cbaf-4286-86a2-0f0404aedb18.png)